app/models/discussion.rb

```diff
@@ -15,14 +15,19 @@ class Discussion < ApplicationRecord
 
   scope :by_language, -> (language) { includes(:exercise).joins(exercise: :language).where(languages: {name: language}) }
   scope :order_by_responses_count, -> (direction) { reorder(validated_messages_count: direction, messages_count: opposite(direction)) }
-  scope :by_requires_moderator_response, -> (boolean) { where(requires_moderator_response: boolean.to_boolean) }
+  scope :by_requires_attention, -> (boolean) { opened_and_requires_moderator_response(boolean).or(pending_review).no_responsible_moderator }
+  scope :opened_and_requires_moderator_response, -> (boolean) { where(requires_moderator_response: boolean.to_boolean)
+                                                                  .where(status: Mumuki::Domain::Status::Discussion::Opened) }
+  scope :no_responsible_moderator, -> { where('responsible_moderator_at < ?', Time.now - MODERATOR_MAX_RESPONSIBLE_TIME)
+                                          .or(where(responsible_moderator_at: nil)) }
+  scope :pending_review, -> { where(status: Mumuki::Domain::Status::Discussion::PendingReview) }
```

Comment:
Is there is a reason for using the FQN instead of just the status symbols?

Reply:
Nah, just an oversight. Fixed on 7dd1e56af0a7f382906f3e706fadff7924cd4a25